### PR TITLE
[RUM-652] Core Web Vitals - FID not being reported on non-chromium browsers

### PR DESCRIPTION
**Pull request by krishnaKapadia**
This PR implements a fix to an issue that we found during testing where non-chromium browsers were not reporting the `First Input Delay` metric.

### The Cause
The `web-vitals` polyfill needs to be includes as close to first as possible as it adds event listeners non-chromium browsers. These listeners need to be available before the `coreWebVitals` window object is initialized.

### The Solution
Changing the order of compilation in the build process so that the polyfill was included before the core raygun script, which inits the `raygunCoreWebVitalFactory`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* v2.22.0
+- Adds the ability to track Core Web Vitals when the `trackCoreWebVitals` option is enabled. This also ensures that First Input Delay is reported from Firefox sessions.
```
Comment (pano-skylakis) [inline CHANGELOG.md:2]: Was it just firefox or all non-chromium based browsers?

File: Gruntfile.js [hunk 2]
```diff
@@ -127,5 +127,5 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['jshint', 'clean', 'concat', 'string-replace', 'uglify']);
 
-  grunt.registerTask('default', ['compile']);
+  grunt.registerTask('default', ['watch', 'compile']);
```
Comment (pano-skylakis) [inline Gruntfile.js:130]: What does adding the watch task alongside compile do here?

**Review by pano-skylakis — APPROVED** [review]
Looks good - had a couple of comments but nothing major.

We'll just have to thoroughly test that changing the order doesn't impact any other dependencies 👍

File: Gruntfile.js (hunk 2 above)
Comment (krishnaKapadia) [reply Gruntfile.js:130]: Was used during testing to speed up development, have removed this 👍

File: CHANGELOG.md (hunk 1 above)
Comment (krishnaKapadia) [reply CHANGELOG.md:2]: It was for non-chromium based browsers, will update the comment now 👍

**Pull request merged**